tests/micro/zephyr/test_zephyr.py

```diff
@@ -608,5 +608,41 @@ def test_schedule_build_with_cmsis_dependency(workspace_dir, board, microtvm_deb
     assert "CMSIS-NN/Include" in cmake_content
 
 
+@tvm.testing.requires_micro
+def test_debugging_enabled(workspace_dir):
+    """Test debugging enabled for LED.
+    We used a physical board(nucleo_l4r5zi) instead of QEMU since
+    LED config is not available on QEMU.
```

Comment:
maybe reference in comment that `verbose` equal to True is what is being tested, it's not super obvious that maps to "debugging enabled"

Reply:
done